packages/kit/src/views/Market/MarketDetailV2/components/SwapPanel/components/ActionButton.tsx

```
@@ -212,8 +233,21 @@ export function ActionButton({
+          ? undefined
+          : (event: GestureResponderEvent) => {
+              if (noAccount) {
+                navigation.pushModal(EModalRoutes.OnboardingModal, {
+                  screen: EOnboardingPages.GetStarted,
+                });
+                return;
+              }
+              onPress?.(event);
+            }
+      }
+      {...buttonStyleProps}
```

Biome (2.1.2)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)